tests/base.py

```diff
@@ -434,8 +431,8 @@ def expected_metadata(self):
             'Holiday': default,
             'IPAddressRange': default,  # new
             'Idea': default,
-            'IdentityProviderEventStore': incremental_created_date,  # new
-            'IdentityVerificationEvent': incremental_created_date,  # new
+            'IdentityProviderEventStore': default_full,  # new
+            'IdentityVerificationEvent': default_full,  # new
```

Comment:
please provide link to the documentation of the streams that have changed replication methods

Reply:
There is no documentation link for replication of these objects. Only when we run sync mode will we be able to know this behavior.